### PR TITLE
Find My: fixes for the next v2.4 minor releases

### PR DESCRIPTION
**Pull request by kapi-no**
Updated Find My and Zephyr revision to include fixes for the v2.4.0 release of the Find My add-on for nRF Connect SDK that will be part of the next minor release.

Ref: NCSDK-22417

### CODE REVIEW

**Review by shanthanordic — APPROVED** [review]
it is ok to merge this PR with lwm2m and thingy91 failures. However I would need +1 from @rafpelc  for RPC failures

**Comment by rafpelc** [comment]
RPC failures not related to PR, can be merged

**Review by rafpelc — APPROVED** [review]

File: west.yml [hunk 1]
```diff
@@ -59,7 +59,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: b57a7488aa3a49fe71b9a042a3d54f2d463b352b
+      revision: pull/1223/head
```
Comment (rlubos) [inline west.yml:62]: `75f67930b8d5616b5c0745de03b14a45fec897ac`

Comment (kapi-no) [reply west.yml]: changed

**Review by mkapala-nordic — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Comment by kapi-no** [comment]
CC: @miha-nordic

**Pull request merged**